src/ensembl/src/content/app/species/components/species-main-view/SpeciesMainView.scss

```diff
@@ -69,29 +69,23 @@
 
     .unit {
       color: $dark-grey;
-      padding-left: 5px;
+      padding: 0 6px;
     }
   }
 
   .summaryStat + .summaryStat {
     grid-column: second_summary_stat;
-
-    .value {
-      font-size: 22px;
-    }
-
-    .unit {
-      color: $dark-grey;
-      padding-left: 5px;
-    }
   }
 
   .exampleLink {
     grid-column: example-link;
   }
-}
-
 
+  .questionButton {
+    top: 2px;
+    position: relative;
+  }
```

Comment:
The `.questionButton` class is also added to the `SpeciesStats.scss` file. Is one of these two class declarations redundant?

Reply:
they are for different components. I didnt know if importing the whole file is better than adding this little css in.